qiskit_experiments/calibration_management/update_library.py

```diff
@@ -151,6 +151,7 @@ def update(
         calibrations: BackendCalibrations,
         exp_data: ExperimentData,
         result_index: Optional[int] = None,
+        parameter: str = None,
```

Comment:
Now that updater classes are more important it would be good to improve the API documentation of the BaseUpdater class (either in that class doc string, or the cal management module docstring) to explain to developers how to subclass a calibration updater class for their experiments. This can be done as followup PR